scripts/ipfs/install

```diff
@@ -20,40 +20,27 @@ rm -rf "$BASE_DIR/tmp"
 # Initialize IPFS
 ipfs init
 
-# Configure systemd to start ipfs.service on system boot
-sudo cp "$BASE_DIR/ipfs.service" /etc/systemd/system/ipfs.service
-sudo sed -i "s|__USER_HOME__|${HOME}|" /etc/systemd/system/ipfs.service
-sudo systemctl daemon-reload
-sudo systemctl enable ipfs.service
-sudo systemctl start ipfs.service
-
 # Configure HTTP to IPFS gateway
 source "$BASE_DIR/../shared/nginx/install"
 sudo cp "$BASE_DIR/ipfs-http-gateway.conf" /etc/nginx/site-path-enabled/ipfs-http-gateway.conf
 sudo systemctl restart nginx.service
 
 source "$BASE_DIR/../shared/nodeinfo/install"
 sudo cp $BASE_DIR/nodeinfo-ipfs /opt/tomesh/nodeinfo.d/ipfs
-sudo sed -i "s|__USER_HOME__|${HOME}|" /opt/tomesh/nodeinfo.d/ipfs
 
 # add cjdns bootstrap address
 ipfs bootstrap add "$IPFS_PEER"
 
-# Service to bootstrap neighboring cjdns nodes if no other ipfs is found
-
 # Download dependencies
 sudo apt-get install -y jq
 
 # Move file
-sudo cp $BASE_DIR/peer-ipfs-bootstrap/check-and-add-peers.sh /usr/local/bin/
-sudo chmod +x /usr/local/bin/check-and-add-peers.sh
-
-# Create file for data
-sudo mkdir /var/lib/peer-ipfs-bootstrap
-sudo touch /var/lib/peer-ipfs-bootstrap/peers.data
-sudo chmod 777 /var/lib/peer-ipfs-bootstrap/peers.data
+sudo cp $BASE_DIR/ipfs-swarm-cjdns.sh /usr/local/bin/
+sudo chmod +x /usr/local/bin/ipfs-swarm-cjdns.sh
 
-# Enable service
-sudo cp $BASE_DIR/peer-ipfs-bootstrap/peer-ipfs-bootstrap.service /etc/systemd/system
+# Configure systemd to start ipfs.service on system boot
+sudo cp "$BASE_DIR/ipfs.service" /etc/systemd/system/ipfs.service
+sudo sed -i "s|__USER_HOME__|${HOME}|" /etc/systemd/system/ipfs.service
```

Comment:
Totally a style thing, but I've been doing stuff like:

```bash
sed -i "s|__USER_HOME__|${HOME}|" "$BASE_DIR/ipfs.service.template" | sudo tee /etc/systemd/system/ipfs.service
```

Which can also be written as 

```bash
sed -i "s|__USER_HOME__|${HOME}|" "$BASE_DIR/ipfs.service.template" | sudo dd of=/etc/systemd/system/ipfs.service
```

If you don't care about tee copying to stdout

Reply:
Looks more complicated? or am I missing something.

Reply:
I understand what you mean now, and yes i agree with you. This was a poorly coded.

I will make an issue.